fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/eventmappings/FixityEvent.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.observer;
+
+import java.util.HashMap;
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+/**
+ * Created by ermadmix on 4/2/2015.
+ * @author ermadmix
+ */
+public class FixityEvent {
```

Comment:
Then you should add a useful constructor to `FedoraEvent` and remove the dependencies. That would be much clearer than introducing a completely new type that has no explicit relationship to any of the eventing machinery.